crates/symbolicator-service/Cargo.toml

```diff
@@ -33,13 +33,13 @@ moka = { version = "0.10", features = ["future"] }
 parking_lot = "0.12.0"
 regex = "1.5.5"
 reqwest = { version = "0.11.0", features = ["gzip", "json", "stream", "trust-dns"] }
-sentry = { version = "0.29.2", features = ["tracing"] }
+sentry = { version = "0.30.0", features = ["tracing"] }
 serde = { version = "1.0.137", features = ["derive", "rc"] }
 serde_json = "1.0.81"
 serde_yaml = "0.9.14"
 sha2 = "0.10.6"
 sourcemap = "6.2.1"
```

Comment:
Can we bump `sourcemap` as well while we're at it?

Reply:
That happened as part of `cargo update` implicitly as its semver compatible. For some dependencies, I tend to bump directly, for others I’m kinda 🤷🏻‍♂️

Reply:
Ah I see in cargo.lock now.